pkg/execution/executor/executor.go

```
@@ -2514,6 +2518,10 @@ func (e *executor) RetrieveAndScheduleBatch(ctx context.Context, fn inngest.Func
 		return err
 	}
 
+	if err := e.batcher.TrackBatchStart(ctx, payload.AccountID); err != nil {
+		return fmt.Errorf("could not track batch start: %w", err)
```

minor: should we return this error or just log it?